fe-nc-news/src/utils/GetComments.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const getComments = (article_id) => {
```

Comment:
You do not really need a separate file for all api calls, there could be hundreds in one application - I would recommend refactoring this into one api.js file in your utils. 
You can export multiple things from one file using ES6 modules - check out [this section of our notes app](https://notes.northcoders.com/courses/js-front-end/react-intro#es6-modules)